extensions/2.0/Vendor/EXT_lights_ies/README.md

```diff
@@ -0,0 +1,126 @@
+| `light` | Index of the light profile. | :white_check_mark: Yes |
+| `multiplier` | Non-negative factor to scale the light's intensity. | No, Default: `1.0` |
+| `color` | RGB value for the light's color in linear space. | No, Default: `[1.0, 1.0, 1.0]` |
+
+## Coordinate System Mapping
+
+The light source measurement sample locations on the spherical photometric web are defined by pairs of vertical and horizontal angles. The zero point `(0.0, 0.0)`, the point at which both angles are zero and the measured light source is generally aimed at, is mapped to the direction `(0.0, 0.0, -1.0)`. That is, the untransformed light points in the direction of the glTF coordinate system's -Z axis.
+
+The mapping of the +X axis depends on the photometric type of the light profile. The following points on the spherical photometric web are in the notation used in `IES LM-75-01/R12` and denote the points to which the +X axis is aimed at for each photometric type:
+
+* Type A: `(0Y, 90X)`
+* Type B: `(90H, 0V)`
+* Type C: `(90V, 270L)`
+
+Lastly, the +Y axis can be computed as the cross product between the +X and -Z axes. 
+
+Thus, the X/Y plane is the horizontal photometric plane and the X/Z plane is the vertical photometric plane for all angular data specified. Note that for light profiles of `Type A` and `Type B` photometry the horizontal angles are specified in clockwise order, while positive rotation in the glTF coordinate system is counter-clockwise.
+
+## Implementation Details
+
+The following IES light profile fields are not required to be supported:
+
+* Any keyword data listed after the first line and before the `TILT` section.
+* The `TILT` section (`TILT=INCLUDE`, `TILT=<filename>` and related values).
+* Light profiles of `Type A` photometry.
+* `<number of lamps>`, `<lumens per lamp>`, `<units type>`, `<width>`, `<length>`, `<height>`, `<file generation type>`, and `<input watts>`.
+
+Furthermore, implementations should be aware of the following differences between IES standard versions:
+
+* The first line of the IES file is either `IESNA:LM-63-1995`, `IESNA:LM-63-2002`, or `IES:LM-63-2019`, corresponding to the respective standard version.
+* In the standard version `IES LM-63-95`, it is valid for light profiles of `Type C` photometry to have horizontal angles starting at 90 degrees and ending at 270 degrees.
+
+## JSON Schema
+
+* [glTF.EXT_lights_ies.schema.json](schema/glTF.EXT_lights_ies.schema.json)
+* [node.EXT_lights_ies.schema.json](schema/node.EXT_lights_ies.schema.json)
+* [light.schema.json](schema/light.schema.json)
+
+## Known Implementations
+
+* [NVIDIA Omniverse](https://www.nvidia.com/en-us/omniverse/)
+
+## Resources
+
+* [IES LM-63-19 (IES Standard File Format for Electronic Transfer of Photometric Data and Related Information), IES Computer Committee](http://www.iesna.org)
+* [IES LM-75-01/R12 (Goniophotometer Types and Photometric Coordinates), IES Computer Commitee](http://www.iesna.org)
```

Comment:
The first link could directly point to the standard's page
- https://store.ies.org/product/lm-63-19-approved-method-ies-standard-file-format-for-the-electronic-transfer-of-photometric-data-and-related-information/.

The second reference should likely be (please confirm)
- https://store.ies.org/product/lm-75-19-approved-method-guide-to-goniometer-measurements-and-types-and-photometric-coordinate-systems/